frontend/.gitignore

```diff
@@ -0,0 +1,3 @@
+/node_modules
+package-lock.json
+tsconfig.json
```

Comment:
I believe that this is a necessary item for this to run and doesn't contain any sensitive information.

Reply:
Thanks for the feedback. I'll add that in! (I wasn't sure which files to leave out).

Reply:
I added the tsconfig.

Reply:
Awesome, thanks!